fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -577,7 +578,7 @@ protected void addLinkAndOptionsHttpHeaders(final FedoraResource resource) {
      * @param relation the relation string.
      * @return the string version of the link.
      */
-    private static String buildLink(final String linkUri, final String relation) {
+    public static final String buildLink(final String linkUri, final String relation) {
```

Comment:
With the intention of only opening access as much as necessary, this can be `protected` instead of `public`.